src/main/java/step2/domain/God.java

```diff
@@ -0,0 +1,29 @@
+package step2.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.IntStream;
+
+public class God {
+
+    public static List<Integer> issueNumbers(int upperbound, int count) {
+        if (count > upperbound) throw new IllegalArgumentException();
+        List<Integer> generatedNumbers = generateNumbers(upperbound);
+        Collections.shuffle(generatedNumbers);
+
+        List<Integer> chosenNumbers = new ArrayList<>();
+        IntStream.range(0, count)
+                .forEach(val -> chosenNumbers.add(generatedNumbers.get(val)));
```

Comment:
```suggestion
        List<Integer> chosenNumbers = IntStream.range(0, count)
                .mapToObj(generatedNumbers::get)
                .collect(Collectors.toList());
```

함수형에서 forEach 를 사용하면, 기존의 절차지향 프로그래밍의 for 문에 비해서 아무런 장점도 없다고 저는 생각합니다.
(성능이 되었든, 가독성이 되었던 기존의 for 문이 더 좋다고 생각합니다. 물론 이건 제 취향이라서 사람마다 다를 수 있습니다.)
forEach 문만이 사용되었다면, 기존의 자바 문법을 사용해주시면 감사하겠습니다.

다만 저도 함수형으로 프로그래밍 하는 것을 좋아하는지라 살짝 suggestion 을 추가했습니다.
(forEach 문을 사용하지 않도록 바꾸었지만, 여전히 비효율적이라서 개선이 필요한 버전입니다.)
제가 보기에 여기서 좀더 함수형을 활용하면, generateNumbers 메서드를 아예 없애버릴 수도 있을 듯한데,
한번 도전해보시기 바랍니다!

Reply:
어떻게 구현해야 할 지 감이 잘 안와서 인터넷에 있는 코드를 활용하였습니다

Reply:
제가 위에 예시로 들어준 generateNubmers 메서드 함수와,
chosenNumbers 에 사용된 함수를 하나로 합성하는 작업이 어렵다면,
IntStream 없이 평범한 for 루프로 구현해주세요.

> 어떻게 구현해야 할 지 감이 잘 안와서 인터넷에 있는 코드를 활용하였습니다.

God.java 가 어떻게 바뀌었는지 확인이 어려운데, 혹시 바뀐 부분이 있다면 해당하는 커밋이 무엇인지 알려주세요 ㅎㅎ

Reply:
Refact(step2): apply code review points에서 God를 LottoNumberGenerator로 수정하였습니다.

커밋을 한번에 모아서 날리는 습관을 이번 기회에 고치도록 하겠습니다ㅠㅠ